handler-proxy/src/main/java/io/netty/handler/proxy/HttpProxyHandler.java

```diff
@@ -74,6 +75,18 @@ public HttpProxyHandler(SocketAddress proxyAddress, String username, String pass
         authzBase64.release();
     }
 
+    /**
+     *
+     * @return endpoint http/ws support
+     */
+    public boolean isTransparent() {
+        return isTransparent;
+    }
+
+    public void setTransparent(boolean transparent) {
+        isTransparent = transparent;
```

Comment:
Could we just add a constructor parameter instead of making `transparent` mutable?